webapps/src/main/webapp/vue-app/taskDrawer/components/TaskDrawer.vue

```diff
@@ -196,16 +197,20 @@
         </div>
       </template>
     </exo-drawer>
-    <task-comments-drawer
-      ref="taskCommentDrawer"
-      :task="task"
-      :comments="comments"
-      @confirmDialogOpened="isDrawerClose = false"
-      @confirmDialogClosed="isDrawerClose = true" />
-    <task-changes-drawer
-      ref="taskChangesDrawer"
-      :task="task"
-      :logs="logs" />
+    <template v-if="drawer">
+      <task-comments-drawer
+        ref="taskCommentDrawer"
+        :task="task"
+        :comments="comments"
+        @confirmDialogOpened="isDrawerClose = false"
+        @confirmDialogClosed="isDrawerClose = true" />
+      <task-changes-drawer
```

Comment:
We should add a dedicated changelog for Task Images attachments to audit Task changes as made for "Task Description" change.

Reply:
ok, thanks, I mark this as resolved and non-blocker.